napari/_vispy/utils_gl.py

```diff
@@ -104,3 +104,38 @@ def fix_data_dtype(data):
                 )
             )
         return data.astype(dtype)
+
+
+BLENDING_MODES = {
+    'opaque': dict(preset='opaque'),
+    'translucent': dict(preset='translucent'),
+    'translucent_no_depth': dict(
+        depth_test=False,
+        cull_face=False,
+        blend=True,
+        blend_func=('src_alpha', 'one_minus_src_alpha', 'zero', 'one'),
+    ),
+    'additive': dict(preset='additive'),
+}
+
+# BLENDING_MODES = {
+#     'opaque': dict(depth_test=True, cull_face=False, blend=False),
+#     'translucent': dict(
+#         depth_test=True,
+#         cull_face=False,
+#         blend=True,
+#         blend_func=('src_alpha', 'one_minus_src_alpha', 'zero', 'one'),
+#     ),
+#     'translucent_no_depth': dict(
+#         depth_test=False,
+#         cull_face=False,
+#         blend=True,
+#         blend_func=('src_alpha', 'one_minus_src_alpha', 'zero', 'one'),
+#     ),
+#     'additive': dict(
+#         depth_test=False,
+#         cull_face=False,
+#         blend=True,
+#         blend_func=('src_alpha', 'one'),
+#     ),
+# }
```

Comment:
```suggestion
```

Beyond removing this commented section, the PR looks good to me, but I'll wait for someone more familiar to reviw.